api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -4429,3 +4440,106 @@ message LoginRuleDelete {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// SAMLIdPAuth is emitted when a user has attempted to authorize against the SAML IdP.
+message SAMLIdPAuth {
+  // Metadata is common event metadata
+  Metadata Metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // User is common user event metadata
+  UserMetadata User = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // SessionMetadata is common event session metadata
+  SessionMetadata Session = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // Status indicates whether the SAML IdP authentication was successful.
+  Status Status = 4 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+}
+
+// SAMLIdPServiceProviderCreated is emitted when a service provider has been added.
+message SAMLIdPServiceProviderCreated {
```

Comment:
Similar to my previous comment, I see that we use `Create` as a suffix rather than `Created`. Same with `Updated` and `Deleted` below.

Reply:
Huh, for some reason I thought I saw `Created` elsewhere, but you're right. I'll change this.